tests/expectations/tests/issue-584-stylo-template-analysis-panic.rs

```diff
@@ -40,14 +40,11 @@ impl Clone for A {
     fn clone(&self) -> Self { *self }
 }
 #[repr(C)]
-#[derive(Debug, Copy, Clone)]
+#[derive(Debug, Default, Copy, Clone)]
 pub struct e<c> {
     pub d: RefPtr<c>,
     pub _phantom_0: ::std::marker::PhantomData<::std::cell::UnsafeCell<c>>,
 }
-impl <c> Default for e<c> {
-    fn default() -> Self { unsafe { ::std::mem::zeroed() } }
-}
```

Comment:
That sounds fine off-hand... In general note that deriving default is not on by default, so it's less likely to break stuff. As long as we keep it working and consistent I think i'd be fine with it.